### DISCUSSION TITLE
Repository F-droid signature is wrong

### DISCUSSION THREAD

**Discussion by AkechiShiro** [bromite/bromite#1907] [Q&A] [answered]
F-droid states that fdroid.bromite.org/froid/repo: Repository was not signed correctly: Supplied certificate fingerprint does not match !

Is anyone else having this issue, something should be updated, right ?

**Comment by HuckToohey** [comment]
This could be the reason I've been getting force-closes on app start. I updated from here, overwriting the F-Droid version, and that solved the crashes for me.

**Reply by csagan5** [reply]
The repository signature verification is unrelated to the app signature verification and you can never overwrite apps with different signatures.

**Comment by csagan5** [comment, marked as answer]
If you supply the wrong signature when adding the repository then you will get that error.

**Reply by AkechiShiro** [reply]
So F-Droid doesn't force the user to input the signature and doesn't warn much about that too, it is just optional, I assumed somehow that there would be a fetching of the signature for 3rd-party repos that would support that, I was very wrong and naive about that thanks for your clarification.

The signature is up-to-date and valid, should I edit the name of this discussion to "Mistakenly thought Repo signature is wrong?"
